client/src/partials/primary_cash/expense/cash_return.js

```diff
@@ -14,6 +14,9 @@ angular.module('bhima.controllers')
   '$q',
   'uuid',
   function ($scope, $routeParams, $translate, $location, messenger, validate, appstate, Appcache, connect, util, exchange, $q, uuid) {
+    /**
+      * TODO use controller alias as possible and redirect url when no daily exchange rate
```

Comment:
Do you want to fix this TODO before this PR is merged?  It seems like you were working with exchange rates and aliases in this PR, so it is the appropriate place for these changes.